server/util/src/canonical_json.rs

```diff
@@ -0,0 +1,120 @@
+pub fn canonical_json(input: &serde_json::Value) -> String {
```

Comment:
Sorry to go over this again, but as per earlier comment (back in the day), did we agree that this indeed is needed ? I remember my last position was to just keep the test ?

btw this test passes

```rust
#[test]
    fn test_canonical_json() {
        let base = json!({
            "A": "string",
            "B": 1.23456789 as f32,
            "C": true,
            "D": 5 as i32,
            "E": [1 as i32, 2 as i32, 3 as i32],
            "F": {
                "AA": "str",
                "BB": 1.23 as f32,
            }
        });
        let base2 = json!({
            "C": true,
            "A": "string",
            "D": 5 as i32,
            "E": [1 as i32, 2 as i32, 3 as i32],
            "F": {
                "BB": 1.23 as f32,
                "AA": "str",
            },
            "B": 1.23456789 as f32,
        });
        // test same object with different order result in the same string
        assert_eq!(&base, &base2);
    }

```

Reply:
I think the canonical_json method is useful to highlight what we want to do here, i.e. that it's important to serialize the value in a canonical way.

Reply:
I think test is sufficient, don't see any point of having logic that's already implemented by the underlying library

Reply:
Yes canonical_json() is just a wrapper around the default serde serialization, no custom serialization is used here.